From: Nicolas Chauvet <kwizart@gmail.com>
Date: Wed, 2 Jan 2013 21:44:33 +0000 (+0100)
Subject: Fix build with unreleased FLAC 1.3.x
X-Git-Url: http://git.videolan.org/?p=vlc.git;a=commitdiff_plain;h=607065b410b4768372c204ccdbbca200230fff25

Fix build with unreleased FLAC 1.3.x

The issue was initialy reported here:
http://bugzilla.redhat.com/891123

FLAC upstream has always expected to include <FLAC/ suffix
for included headers as shown in examples/c/decode/file/main.c

The FLAC suffix will not be be made available in the flac.pc
for the next release:
http://lists.xiph.org/pipermail/flac-dev/2012-April/003355.html

Signed-off-by: Jean-Baptiste Kempf <jb@videolan.org>
---

--- a/modules/codec/flac.c
+++ b/modules/codec/flac.c
@@ -38,8 +38,8 @@
 #include <vlc_codec.h>
 #include <vlc_aout.h>
 
-#include <stream_decoder.h>
-#include <stream_encoder.h>
+#include <FLAC/stream_decoder.h>
+#include <FLAC/stream_encoder.h>
 
 #include <vlc_block_helper.h>
 #include <vlc_bits.h>
